package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:DTAConfig:DownloadCompleteText=可选组件 {0} 下载成功。; Download of optional component {0} completed succesfully.
+Client:DTAConfig:DownloadCompleteTitle=下载完成; Download Completed
+Client:DTAConfig:Downloading=下载中...; Downloading...
+Client:DTAConfig:DownloadingText=可选组件正在下载中。如果您退出选项菜单，下载将被取消。@@您确定要继续吗？; Optional component downloads are in progress. The downloads will be cancelled if you exit the Options menu.@@Are you sure you want to continue?
+Client:DTAConfig:DownloadingTitle=下载进行中; Downloads in progress
+Client:DTAConfig:Enable=启用; Enable
+Client:DTAConfig:ForceUpdate=强制更新; Force Update
+Client:DTAConfig:ForceUpdateConfirmText=警告：强制更新将导致文件被重新验证和重新下载。虽然这可能修复游戏文件的问题，但也可能删除对此安装进行的某些自定义修改。请自行承担风险！@@如果继续，选项窗口将关闭，客户端将开始检查更新。@@您真的要强制更新吗？; WARNING: Force update will result in files being re-verified@and re-downloaded. While this may fix problems with game@files, this also may delete some custom modifications@made to this installation. Use at your own risk!@@If you proceed, the options window will close and the@client will proceed to checking for updates.@@Do you really want to force update?
+Client:DTAConfig:ForceUpdateConfirmTitle=强制更新确认; Force Update Confirmation
+Client:DTAConfig:FriendsOnly=只接收好友的游戏邀请; Only receive game invitations from friends
+Client:DTAConfig:FullscreenClient=全屏客户端; Fullscreen Client
+Client:DTAConfig:HotkeyNone=无; None
+Client:DTAConfig:InGameResolution=游戏内分辨率：; In-game Resolution:
+Client:DTAConfig:Install=安装; Install
+Client:DTAConfig:IntegerScaledClient=整数缩放客户端; Integer Scaled Client
+Client:DTAConfig:IntegerScaledClientToolTip=为客户端启用整数缩放。这将使客户端使用最接近的合适分辨率以保持图形清晰，但代价是在某些分辨率下可能会出现黑边。@@此外，启用此选项还将允许调整客户端窗口大小（不影响所选的客户端分辨率）。; Enable integer scaling for the client. This will cause the client to use@the closest fitting resolution that is required to maintain sharp graphics,@at the expense of black borders that may appear at some resolutions.@@Additionally, enabling this option will also allow the client window @to be resized (does not affect the selected client resolution).
+Client:DTAConfig:Language=语言：; Language:
+Client:DTAConfig:MainMenuMusic=主菜单音乐; Main menu music
+Client:DTAConfig:MoveDown=下移; Move Down
+Client:DTAConfig:MoveUp=上移; Move Up
+Client:DTAConfig:MusicVolume=音乐音量：; Music Volume:
+Client:DTAConfig:NewHotKey=新快捷键：; New hotkey:
+Client:DTAConfig:NoGameLobbyMessageAudio=游戏运行时不播放大厅消息音效; Don't play lobby message audio when game is running
+Client:DTAConfig:NoLobbiesMusic=在大厅中不播放主菜单音乐; Don't play main menu music in lobbies
+Client:DTAConfig:NotAvailable=不可用; Not Available
+Client:DTAConfig:OptionalComponentDownloadFailedText=可选组件 {0} 下载失败。@详情请见 client.log。@@如果此问题持续存在，请联系您的 Mod 作者以获取支持。; Download of optional component {0} failed.@See client.log for details.@@If this problem continues, please contact your mod's authors for support.
+Client:DTAConfig:OptionalComponentDownloadFailedTitle=可选组件下载失败; Optional Component Download Failed
+Client:DTAConfig:PingUnofficial=检测非官方 CnCNet 线路; Ping unofficial CnCNet tunnels
```

You are using the word `线路` for tunnel while most of your translation texts use `专线`

```suggestion
Client:DTAConfig:PingUnofficial=检测非官方 CnCNet 专线; Ping unofficial CnCNet tunnels
```